src/main/java/edu/hm/hafner/analysis/registry/VeracodePipelineScannerDescriptor.java

```diff
@@ -0,0 +1,34 @@
+package edu.hm.hafner.analysis.registry;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.VeracodePipelineScannerParser;
+
+import static j2html.TagCreator.*;
+
+/**
+ * A descriptor for Veracode Pipeline Scanner.
+ *
+ * @author Juri Duval
+ */
+class VeracodePipelineScannerDescriptor extends ParserDescriptor {
+    private static final String ID = "veracode-pipeline-scanner";
+    private static final String NAME = "Veracode Pipeline Scanner";
+
+    VeracodePipelineScannerDescriptor() {
+        super(ID, NAME);
+    }
+
+    @Override
+    public IssueParser createParser(final Option... options) {
+        return new VeracodePipelineScannerParser();
+    }
+
+    @Override
+    public String getHelp() {
+        return join(text("Use commandline"),
+                code("java -jar pipeline-scan.jar --json_output=true --json_output_file=results.json"),
+                text(", see"),
+                a("Veracode Pipeline Scanner").withHref("https://docs.veracode.com/r/c_about_pipeline_scan"),
```

Comment:
Can you use this URL in `getUrl` as well?

Reply:
And maybe an additional link to https://github.com/jenkinsci/veracode-scan-plugin/blob/master/src/main/webapp/icons/veracode-48x48.png in `getIcon`. (I did not find an SVG?)

Reply:
Good suggestion. I added `getUrl` and  `getIcon` now

Will raise a separate PR to add those for Aqua scanner as well.